UKHomeOffice/cop-ui
# COP-10340: Adjust Home Page UI and target page width

### Description
The targets list page now has a width of 1400px, while all of the other pages have the new width of 1160px.

The Home Page styling has been adjusted to reduce the glare that had been introduced in the previous iteration.

https://support.cop.homeoffice.gov.uk/browse/COP-10340

### To test
The styling of the Home Page should look like the current Figma files (see the ticket) and the width of the Targets list should now switch to 1400px, while the rest of the application should remain at 1160px.
